bot/moderation/moderation.py

```diff
@@ -64,7 +64,7 @@ async def modmail(self, ctx: commands.Context):
             await ctx.author.send("Deine Nachricht wurde erfolgreich an die Moderatoren weitergeleitet!\n"
                                   "__Hier deine Bestätigung:__", embed=embed_confirmation)
 
-    @modmail.command(name='get')
+    @modmail.command(name='get', hidden=True)
```

Comment:
what does hidden do?

Reply:
Hide the command in every help message generated by the library.
So if someone writes `!help modmail` he won't see that this subcommand even exists.